src/main/java/com/feelmycode/parabole/dto/CouponUserResponseDto.java

```diff
@@ -27,16 +26,12 @@ public class CouponUserResponseDto {
 
         public CouponUserResponseDto(Coupon coupon, UserCoupon userCoupon, String sellerName) {
 
-                // 동일한 couponId 로 물려있는 coupon 과 userCoupon 인 경우에
                 this.name = coupon.getName();
                 this.serialNo = userCoupon.getSerialNo();
                 this.sellerName = sellerName;
-                this.type = coupon.getType().ordinal();
-                if (type == 1) {
-                        this.RateOrAmount = coupon.getDiscountRate();
-                } else if (type == 2) {
-                        this.RateOrAmount = coupon.getDiscountAmount();
-                }
+                this.type = coupon.getType().getName();
+                if (type.equals("RATE")) this.RateOrAmount = coupon.getDiscountRate();
```

Comment:
if문에 항상 중괄호로 감싸주세요
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf
12페이지 참고하시면 좋아용
```suggestion
                if (type.equals("RATE")) {
                    this.RateOrAmount = coupon.getDiscountRate();
                }
```

Reply:
넵 컨벤션 공유 감사용

Reply:
반영완료